lib/apolloClient.ts

```diff
@@ -0,0 +1,28 @@
+import { ApolloClient, createHttpLink, InMemoryCache } from '@apollo/client';
+import { setContext } from '@apollo/client/link/context';
+
+const httpLink = createHttpLink({
+  uri: process.env.uri,
```

Comment:
please rename `uri` it's too generic, something like reflow_graphql_endpoint etc...